VCFv4.3.tex

```diff
@@ -577,43 +591,113 @@ \section{INFO keys used for structural variants}
 
 \footnotesize
 \begin{verbatim}
-##INFO=<ID=SVLEN,Number=.,Type=Integer,Description="Difference in length between REF and ALT alleles">
+##INFO=<ID=SVLEN,Number=A,Type=Integer,Description="Difference in length between REF and ALT alleles">
 \end{verbatim}
 \normalsize
-One value for each ALT allele. Longer ALT alleles (e.g.\ insertions) have positive values, shorter ALT alleles (e.g.\ deletions) have negative values.
+
+One value for each ALT allele.
+Longer ALT alleles (e.g.\ insertions) have positive values, shorter ALT alleles (e.g.\ deletions) have negative values.
+Inversion events should have $SVLEN=0$ since the total length of the allele matches the reference.
+The missing value should be used for inter-chromosomal $BND$ records.
```

Comment:
I'd clarify whether it's required to include `SVLEN` for `BND` records, and also explicitly state what `SVLEN` should be for intra-chromosomal `BND` records.

Reply:
`BND`s are a whole separate world of pain, so I've extracted this comment into #546. For the scope of this PR, marking as resolved.